docs/Developers/Development_Setup.rst

```diff
@@ -52,6 +52,10 @@ Using virtualenv
         $ virtualenv coala-venv
 
 NOTE:
+Depending on your system,you most likely need :code:` sudo ` to install virtualenv.
+::
```

Comment:
Running `coala` locally would flag this, and you can fix gitmate stuff before pushing ;)